setup.py

```diff
@@ -77,23 +77,23 @@
         'entrypoints>=0.3',
         'jinja2>=2.11',
         'jsonschema>=3.2.0',
-        'jupyter_core>=4.0,<5.0',
+        'jupyter_core>=4.0',
         'jupyter_client>=6.1.7',
         'jupyter_server>=1.7.0',
         'jupyterlab>=3.0.17',
         'jupyterlab-git~=0.32',
         'jupyterlab-lsp>=3.8.0',
         'jupyter-resource-usage>=0.5.1',
-        'minio>=7.0.0,<8.0.0',
+        'minio>=7.0.0',
         'nbclient>=0.5.1',
         'nbconvert>=5.6.1',
         'nbdime~=3.1',
         'nbformat>=5.1.2',
         'networkx>=2.5.1',
         'papermill>=2.1.3',
         'python-lsp-server[all]>=1.1.0',
-        'pyyaml>=5.3.1,<6.0',
-        'requests>=2.25.1,<3.0',
+        'pyyaml>=5.3.1',
+        'requests>=2.25.1',  # Note: future major releases of `requests` are expected to include breaking changes
```

Comment:
Is there a specific source for this statement that makes it necessary to state this explicitly?

Reply:
That was my source as well, but I was misreading it -- I kept substituting the phrase 'backwards incompatibility' with breaking change 🙄 . I'll go ahead and remove!